sql/src/SqlEval.sk

```diff
@@ -1683,6 +1683,21 @@ fun evalWithContext(
   evaluator.stmt(context, params, stmt, inTransaction);
 }
 
+fun sqlPurgeFrom(
+  replicationId: Int,
+): (mutable SKStore.Context, SKStore.Dir) ~> SKStore.Tick {
+  (ctx, dir) ~> {
+    table = getSqlTableName(dir.getDirName());
+    ackTable = feedbackTable(table);
+    watermark = getWatermark(ctx, ackTable, Some(replicationId));
+    default = SKStore.Tick(ctx.tick.value - 1000);
```

Comment:
Is there reasoning for this choice of constant or just a best-guess for a reasonable amount to purge?

Also, I _think_ that everything should be fine with negative ticks, but am not 100% confident on that -- are you? If not, maybe we could take the max with 0 here

Reply:
This has been the logic forever, so I'm pretty confident. Julien picked 1k out of thin air to be honest. We need to do some measurement at some point.

Note this just sets the tombLimit for a dir. We don't end up with negative ticks in the data. We also have a check during purge that the limit is monotonic.